pilot/spec/build_manager_spec.rb

```diff
@@ -1,10 +1,18 @@
+require 'active_support'
+require 'active_support/core_ext'
```

Comment:
What extensions do we actually use and need of these?
Why only in the tests?

Reply:
Oops! You are absolutely right, these should not go in the tests. The reason why it is included is for the mb_chars method that is used in build_manager.rb (not in the spec). It is used to truncate the changelog string by bytes and not characters.

So I should actually move the require to build_manager.rb. It was a mistake adding it to the spec but it worked because I launched the test and thus the require was loaded from the spec when using the actual method.